action/protocol/staking/staking_statereader.go

```diff
@@ -99,6 +206,9 @@ func (c *compositeStakingStateReader) readStateCandidates(ctx context.Context, r
 	}
 
 	// add liquid stake votes
+	if !protocol.MustGetFeatureCtx(ctx).AddLSDVotes {
+		return candidates, height, nil
+	}
```

Comment:
if this result is NOT used by slasher (write into its own namespace, become part of consensus), no need to add this check

Reply:
confirm if this is necessary for being consistent with `ActiveCandidates`

Reply:
After communicating with Zong Zheng and Chen Zhen, it was clarified that the stake webpage does not utilize `readstate` to retrieve votes; instead, it reads from the analyser database.

Reply:
btw, analyser also need to support iip-13